modules/integration_azure-functions/README.md

```diff
@@ -95,15 +97,19 @@ with this Terraform [module](https://github.com/claranet/terraform-signalfx-inte
 Here is the list of required metrics for detectors in this module.
 
 * `AppConnections`
+* `azure.function.errors`
 * `FunctionExecutionCount`
 * `Http5xx`
 * `Threads`
 
 
+## Notes
 
+If you want to enable azure.function.errors detector, you will need to instrusment your function. The wrapper for C#, Java and NodeJs is provided by SignalFX. But for Python, you will have to use [signalfx-azure-function-python](https://github.com/claranet/signalfx-azure-function-python)
```

Comment:
may be can you list available repos for other languages too?

Reply:
Now listed in source_doc